po/eo.po

```diff
@@ -0,0 +1,236 @@
+# Esperanto translation for Ladybird.
+msgid ""
+msgstr ""
+"Project-Id-Version: ladybird wip\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-08-27 00:38+0300\n"
+"Language: eo\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 "
+"&& n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
+
+#: window.ui:67
+msgid "...will appear here, once we implement them :^)"
+msgstr "...aperos ĉi tie, post kiam ni realigos ilin :^)"
+
+#: window.ui:166 window.ui:296
+msgid "Bookmarks"
+msgstr "Legosignoj"
+
+#: window.ui:66
+msgid "Bookmarks & History"
+msgstr "Legosignoj kaj Historio"
+
+#: window.ui:55
+msgid "Close _Other Tabs"
+msgstr "Fermi _Aliajn Langetojn"
+
+#: shortcuts-dialog.ui:83
+msgid "Close current tab"
+msgstr "Fermi aktualan langeton"
+
+#: window.ui:148
+msgid "Enter a URL to open..."
+msgstr "Entajpi URL..."
+
+#: Window.cpp:140
+msgid "Failed to pick file"
+msgstr "Malsukcesis elekti dosieron"
+
+#: shortcuts-dialog.ui:66
+msgid "Focus location entry"
+msgstr "Enfokusigi lokan enskribigon"
+
+#: shortcuts-dialog.ui:10
+msgid "General"
+msgstr "Ĝenerala"
+
+#: Application.cpp:65
+msgid ""
+"Ladybird is a browser based on LibWeb web engine and LibJS JavaScript "
+"engine, developed by a large team of contributors as a part of the "
+"SerenityOS project."
+msgstr ""
+"Ladybird estas retumilo surbaze de la LibWeb retmotoro kaj la LibJS "
+"JavaScript-motoro, estigita de granda teamo kontribuantoj kiel parto "
```

Comment:
I see `Ĝavoskripto` in use instead of JavaScript in [general online use](https://www.reddit.com/r/Esperanto/comments/14i1w5h/eviti_ke_%C4%9Davoskripto_modifu_la_tondujon_en_firefox/), and its used as a translation of JavaScript in both [Epiphany](https://gitlab.gnome.org/GNOME/epiphany/-/blob/master/po/eo.po#L225) and [Konqueror](https://invent.kde.org/network/konqueror/-/blob/master/po/eo/konqueror.po?ref_type=heads#L214), although Epiphany uses it and `JavaScript` [inconsistently](https://gitlab.gnome.org/GNOME/epiphany/-/blob/master/po/eo.po#L3384), but Konqueror uses `Ĝavoskripto` everywhere, so this should probably use that here